src/cli.ts

```diff
@@ -126,6 +131,16 @@ const latestPolkadotReleaseURL = async (
   }
 };
 
+function getTestNameFromFileName(testFile: string): string {
+  const fileWithOutExt = testFile.split(".")[0];
+  const fileName: string = fileWithOutExt.split("/").pop() || "";
+  const parts = fileName.split("-");
+  const name = parts[0].match(/\d/)
+    ? parts.slice(1).join(" ")
+    : parts.join(" ");
+  return name;
+}
```

Comment:
You should probably reuse [the misc util function](https://github.com/paritytech/zombienet/blob/main/src/utils/misc.ts#L115-L128) instead of making a new one

Reply:
This one have an extra logic to use the file name as tests name
e.g `0001-dispute-valid-block.zndsl` return `dispute valid block`.
Thanks!